Application/Source/Detail/View/DetailView.swift

```diff
@@ -3,46 +3,29 @@ import SnapKit
 
 class DetailView: UIView {
 
-    let title: UILabel = {
-        let label = UILabel()
-        label.textAlignment = .center
-        label.numberOfLines = 0
-        return label
+    let title = UILabel()
+    let button = UIButton()
+    let selectionResult = UILabel()
+
+    private let spacingView: UIView = {
+        let view = UIView()
+        view.snp.makeConstraints { $0.height.equalTo(35) }
+        return view
```

Comment:
Rather than having a spacing view, could we have two UIStackViews inside of an outer UIStackView, then just give that UIStackView the appropriate spacing value? If we build it this way, we could even easily use a theme to define the spacing.